config/config.go

```diff
@@ -108,6 +122,18 @@ func (config *MixConfig) LoadDefaultsForPath(path string) {
 	config.Mixer.LocalRPMDir = filepath.Join(path, "local-rpms")
 	config.Mixer.LocalRepoDir = filepath.Join(path, "local-yum")
 
+	// [Os]
+	config.Os.Name = "<NoChange>"
+	config.Os.Version = "<NoChange>"
+	config.Os.ID = "<NoChange>"
+	config.Os.IDLike = "<NoChange>"
+	config.Os.PrettyName = "<NoChange>"
+	config.Os.AnsiColor = "<NoChange>"
+	config.Os.HomeURL = "<NoChange>"
+	config.Os.SupportURL = "<NoChange>"
+	config.Os.BugReportURL = "<NoChange>"
+	config.Os.PrivacyPolicyURL = "<NoChange>"
```

Comment:
Initially I thought of put empty strings to indicate no change, but that might trigger another thought what if the user want to overwrite the value to empty instead of no change.